browser/LocalizationLoader.js

```diff
@@ -30,29 +30,26 @@
 
 'use strict';
 
-module.exports = LocalizationLoader;
+class LocalizationLoader {
 
-/**
- * Creates new instance of localization loader.
- * @param {Window} $window browser window.
- * @constructor
- */
-function LocalizationLoader($window) {
-	this._localization = $window.localization &&
-		typeof ($window.localization) === 'object' ? $window.localization : {};
-}
+	/**
+	 * Creates new instance of localization loader.
+	 * @param {ServiceLocator} locator Locator to resolve dependencies.
+	 * @constructor
```

Comment:
We don't need a `@constructor` annotation anymore, it's obvious that the method is a constructor.